api/kessel/relations/v1beta1/relation_tuples.proto

```diff
@@ -34,6 +34,21 @@ service KesselTupleService {
 			delete: "/v1beta1/tuples"
 		};
 	};
+	rpc ImportBulkTuples (stream
+			ImportBulkTuplesRequest) returns (ImportBulkTuplesResponse) {
+		option (google.api.http) = {
+			post: "/v1beta1/tuples/bulkimport"
+			body: "*"
+		};
+	};
+}
+
+message ImportBulkTuplesRequest {
+	repeated Relationship tuples = 1;
+}
+
+message ImportBulkTuplesResponse {
```

Comment:
My first thought is that it might be difficult to honour num_loaded in ImportBulkTuplesResponse, because spicedb does not give you anything in https://buf.build/authzed/api/docs/main:authzed.api.v1#authzed.api.v1.WriteRelationshipsResponse. If we're bulk updating with touch semantics it may be non trivial to figure out how many relationships were actually "loaded" ("written" might be clearer than "loaded" here).

Reply:
I forgot spicedb had a bulk import api that maps perfectly. Please ignore!